controllers/operatorpolicy_controller.go

```diff
@@ -300,16 +333,16 @@ func buildOperatorGroup(
 ) *operatorv1.OperatorGroup {
 	operatorGroup := new(operatorv1.OperatorGroup)
 
-	gvk := schema.GroupVersionKind{
-		Group:   "operators.coreos.com",
-		Version: "v1",
-		Kind:    "OperatorGroup",
+	if policy.Spec.OperatorGroup == nil {
+		operatorGroup.SetGroupVersionKind(operatorgroupgvk)
```

Comment:
Please use camel case for the `operatorgroupgvk` variable (e.g. `operatorGroupGVK`).